features/step_definitions/journey_test_steps.rb

```diff
@@ -189,11 +189,11 @@
 
 When /I click the '(.*)' link in the '(.*)' column for the supplier '(.*)'/ do |link_name, column_name, supplier_name|
   @supplierName = supplier_name
+  
+  table_row_first_cell = page.first(:css, "td.summary-item-field-first") {|elem| elem.text == supplier_name}
```

Comment:
What does the final `{|elem| elem.text == supplier_name}` do?  Is this a shortcut for [`select`](https://ruby-doc.org/core-2.4.1/Enumerable.html#method-i-select) or does it do something else?